imagery/i.atcorr/create_iwave.py

```diff
@@ -178,25 +168,25 @@ def pretty_print(filter_f):
     Create pretty string out of filter function
     8 values per line, with spaces, commas and all the rest
     """
-    pstring = ""
+    pstring = []
     for i in range(len(filter_f) + 1):
-        if i % 8 is 0:
-            if i is not 0:
+        if i % 8 == 0:
+            if i != 0:
                 value_wo_leading_zero = ("%.4f" % (filter_f[i - 1])).lstrip("0")
-                pstring += value_wo_leading_zero
-            if i > 1 and i < len(filter_f):
-                pstring += ", "
-            if i is not 1:
```

Comment:
The replacement of this doesn't seem exactly the equivalent. Is the new logic expected behavior? (Old one got triggered for -1, 0, 2, 3, 4..., new gets triggered for 2, 3, 4...)

Reply:
You are right. It is not equivalent. However, `i` goes in a `range` here and should not contain negative values -- if a negative value slipped in, it would be a mistake and it should not get in here.